src/quacc/settings.py

```diff
@@ -90,6 +88,14 @@ class QuaccSettings(BaseSettings):
             "Some workflow engines have an option to do this for you already."
         ),
     )
+    COMMON_TMPDIR: str | None = Field(
+        None,
+        description=(
+            "Whether to use a common temporary directory for all jobs."
+            "This is useful for parallel workflows where the same files are needed "
+            "by multiple jobs. Should be a string."
+        ),
+    )
```

Comment:
I am a bit hesitant to add a generic setting like this for several reasons. Let's try not to rely on this if possible.